fix(WW-3713): Multi-language support for breadcrumb item labels

leo91000 opened the pull request.

leo91000 committed:
fix(WW-3713): Multi-language support for breadcrumb item labels

AlexisJC commented on src/wwElement.vue:
```diff
@@ -32,7 +32,9 @@
-                            :ww-props="{ text: item.label }"
+                            :ww-props="{
+                                text: typeof item.label === 'object' ? wwLang.getText(item.label) : item.label,
+                            }"
```
Est ce qi'om faidraot pas juste faire le getText dans la computed breadcrumbItems à la place d'ajouter wwLib.wwLang dans data et le faire dans le template ?

leo91000 replied:
Done

leo91000 committed:
fix(WW-3713): Update binding validation tooltip to document multilang label support

Updated the tooltip to show that label property can accept both string values and multilang objects.
